website/content/en/status/report-2023-04-2023-06/opensearch.adoc

```diff
@@ -0,0 +1,21 @@
+=== OpenSearch
+
+Links: +
+link:https://opensearch.org/[OpenSearch] URL: link:https://opensearch.org/[] +
+
+Contact: OpenSearch Team <opensearch@FreeBSD.org>
+
+OpenSearch is a community-driven, open-source search and analytics suite derived from Apache 2.0 licensed ElasticSearch 7.10.2 & Kibana 7.10.2.
+It consists of a search engine daemon, OpenSearch, and a visualization and user interface, OpenSearch Dashboards.
+OpenSearch enables people to easily ingest, secure, search, aggregate, view, and analyze data.
+These capabilities are popular for use cases such as application search, log analytics, and more.
+With OpenSearch people benefit from having an open-source product they can use, modify, extend, monetize, and resell as they want.
+
+Since our last status report, the OpenSearch Team has regularly updated the OpenSearch and OpenSearch-Dashboards ports to follow upstream releases.
+
+While minor version updates are generally straightforward, some use cases make this slightly harder and require more work.
+For this reason, the OpenSearch Team has recently decided to change the way we package OpenSearch.
+Instead of our current scheme where we provide 2 ports, one for the latest major version (package:textproc/opensearch[], currently at version 2.8.0) and one for the legacy major version in maintenance mode(package:textproc/opensearch13[], currently at version 1.3.10), we will move to distribute each minor version as a new port (i.e. package:textproc/opensearch26[], package:textproc/opensearch27[], package:textproc/opensearch28[] and so on).
+
+Because upstream only provides updates for the latest major version, when a new minor version is made available and a new port is added, the previous minor version will be marked as deprecated, and the minor version before that will have an expiration date set, and will be removed at least three months later.
```

Comment:
@alonsobsd reading https://github.com/opensearch-freebsd/freebsd-ports/pull/1#issuecomment-1597776695 I understand that this is not really necessary and we can stick with the current scheme unless AWS does :hankey: and then we would handle this with a new version-specific port?

If so, this whole PR does not have much to say to our users, and I think we can dismiss it.  What do you think?